style.css

```diff
@@ -329,3 +329,111 @@ p {
   border-top: 1px solid #dfe1e6;
   margin-top: 20px;
 }
+
+footer {
+  display: flex;
```

Comment:
- [ ] [OPTIONAL] For increasing 🆙 your chances in the job search 💼 after finishing microverse, we request that you follow coding best practices 💯  in all of your projects. One of these practices is **DRY** (**D**on't **R**epeact **Y**ourself). That means if you find yourself repeating 🔁 a specific line of code in multiple places, you have to find a way to only use it once 1️⃣. This way when you update ♻️  this line, you won't have to remember where else did you write that to update the other copies.

### You can check this in the [Html & Css Best Practices Page](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)

![image](https://user-images.githubusercontent.com/60979926/164267978-27c9c5e1-37bb-4d2d-81ef-e7cd491bfbdb.png)

### In our case

you can easily use a `.flex` class in multiple places .

- (your code)

```css
.footer {
  /* ...code */
  display: flex;
  /* ...code */
}

.contact-form {
  /* ...code */
  display: flex;
  /* ...code */
}
```

- suggestion

```css
.flex {
  display: flex;
}
```

Reply:
Good job in implementing the changes 👍